otterdog/eclipse-opendut.jsonnet

```diff
@@ -44,11 +44,20 @@ orgs.newOrg('eclipse-opendut') {
           ],
           requires_conversation_resolution: true,
         },
-        orgs.newBranchProtectionRule('development') {
+      ],
+      rulesets: [
+        orgs.newRepoRuleset('protect-development-branch') {
+          allows_updates: true,
           allows_force_pushes: true,
-          required_approving_review_count: null,
+          include_refs+: [
+            "refs/heads/development",
+          ],
           requires_pull_request: false,
-        },
+          required_approving_review_count: null,
+          required_status_checks+: [
+            "build"
```

Comment:
ok that was a misunderstanding from my side, when the build status check is enabled it would mean that the CI workflow would have to pass before merging a feature branch into the development branch in case you open up a PR. If you directly push to the development branch, it would not be taken into account. But I can delete it nevertheless.